voxpupuli-release.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |s|
   s.executables = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
 
   # Runtime dependencies, but also probably dependencies of requiring projects
+  s.add_runtime_dependency 'github_changelog_generator', '>= 1.16.1'
```

Comment:
This is something I why I had doubts to add it to this gem. It ends up pulling in a lot of dependencies while the release was previously neat and small. I'm not sure it should block, but I at least want to share it.

Reply:
I have no objection to reverting the corresponding code in the other PR. For now I added it in a commit: if we change our mind, we can remove it before merging.